app/partials/dashboard/dashboard.html

```diff
@@ -0,0 +1,4 @@
+<div>
```

Comment:
dashboard has not been merged to development yet, after this merge i merge development branch to dashboard (I wrote you about this..)